### ISSUE TITLE
Can't upload public key without comment

### ISSUE THREAD

**Issue by matheweis** [keybase/keybase-issues#1777]
When I tried to upload a public key that has the comment removed, I get an error; something about block not found.

**Comment by zQueal** [comment]
Could you post the exact public key? I don't think I've ever tried to setup an account with a key with no comment before, but the block not found sounds like you're not copying the entire key. Of course I could be wrong.

**Comment by dtiersch** [comment]
the comment is no problem. I use no comment on all identities and had no trouble uploading my pubkey. (In fact there are a lot of people who consider using comments [bad practice](https://www.debian-administration.org/users/dkg/weblog/97))

**Comment by matheweis** [comment]
I agree it is poor practice, which is why I had initially tried to remove my comments... I just tried again and received an error once more. The actual error is "No packets; cannot extract a key"

**Comment by matheweis** [comment]
Here is the original key:

-----BEGIN PGP PUBLIC KEY BLOCK-----
Comment: GPGTools - https://gpgtools.org

mQINBFWcruEBEAC8G5EpiJUJwpuoFVtcks96EKJduiVAoGS4PUyNwPLO8mJ3xhRc
MyVpU+k9Io04rSBt3J1uTo3bPuJszlXK3TQyxrKlQBr6DliFoPwKTyg637wupwcC
MXtdJhWm9O2a1X9YK1jBkApcIy8uXVRa8HqOExatdZLSRfNbE02y8r9wkMo18zK0
eLjziR6jVDoqChLEOYdTQfcudMHjteA3a00oF0jz7mmhMFY+7P0z9nI2KcfrC+WO
zyVyq7FzuGQJem6SzJhCXPvc3A6DH72NXhATVxiPrq6MTRafGnX1EIdZdSofeC3+
RTDGy/IaTqQeFAg6IOQcACff/fAr5JXLdVgjngyjyjMdUX+WftH3ey8hMTA+RtQX
N5T4ABeIb3SJXWGaj5NBa07ReE4Pkbc4Bgqd6l+zkJyJ1X+mTMeBWME+23Ax5ihI
0a6T7RdOBsbynyl67HeDK8h5krcK6nS1fuPPqp7OOXOO+Ks4AzCI9ip8j2PO4rQ4
JHLauMZ4ySTiXFmqZRYtwuQ0KeOi0I3uX+YVWQAX9iZ2YYULSY+q0JPWMeuLuecx
bPqkpCVt1WV9c3i4Or5PG3AG5uLiP9MfqiRX+FObOVUfhkxLxALbI8KcitVyL3/z
m7CvpL98xXIyZKgRnMx1oThwe7kmGrneXhixD87LhlUrYW/xcjYQqCEg8QARAQAB
tB5NYXRoZXcgRWlzIDxtYXRoZXdAZWlzYm94Lm5ldD6JAj0EEwEKACcFAlWcruEC
GwMFCQhwyoAFCwkIBwMFFQoJCAsFFgIDAQACHgECF4AACgkQ2OQ2RZLzp0nqiRAA
jXE7485hc78MUZ6/yl8eZRC4k+4HSrojt4VHoxt3YqBseO5KMOoQSS5ktSEVQ8Jv
G0YZLA1EFuaz5lT2KcJy0EM4zWWSX+MIPtmpG2kP9QdO9h6+Wf3ikjmFuYnLNy8K
Feohsbb3jNR2YiEi86ue+VcSQZkTYPVEqZnPtxLofJw/kTs3ufMP7ZMRxi22PCmc
a1R3rM7+p2TpPScYgF46tGN7hn4RAXOeleuORSBNpyzIj5XiV6conUJZ0/WxY9i8
4ZwYQA0xWP7/fbZBhAo/VCrghcLTI1KbsBcTmZ2J34kdRxOM9G3aRVUoxxnetIy5
Kqfg1G54gDwil6LAnYwcB2msSamDFyzqTQIWsJAbi5Dj1KQCYcZuxIH1jBvBoGIG
XbHwoMT0Zt9CPyxkREJAQ34WnYVl8H0RDEJtZ6u2my2/f+ntcOyJDzI93zTeg7CT
2+KSonBlEHfyr9s4roxh0HCjRTr7UxnH9vrkIx7OhAv8yWgjgn2EJ3LFfMwtOscY
KiCi1kPqNu6cuFxgCp8lh8eDNe10mH0ZK4HbVswD6eJ0GtuZBe1FmSqKZq5s8IOl
aODACEAQ3d493irNGAEUm2YXc4mppNiWw9IxNX8D2jBd5BQYo2Ows5DdzIMQMVwz
hMqpy9WCdPPclCRl8BBaE8vn161yD7KM1W2ARee6PZqIRgQTEQoABgUCVZyvCwAK
CRBa1iNsrJkjSq/PAJ4mTwQou4V7MOy5qimxJAXbqkK8MgCeIqfw+igA14cLVjhN
BhdXGsbfLqy5Ag0EVZyu4QEQAL6U2h/V6TGOrb5OgaIZy+k+ZbxjKm7aQMxEJSkM
t/kAQzfc09g46ugfybxlPa8ZN/+yNnGAJQTvJcsVuJQfoj9AW8VSizXjfB/rBPiy
5ykyXRpjoMxEZMyR9d7kzjwAn5u2N7GPgmAPuJb6syvNxlXhZqwLiTX4f4rXXyz2
iHYb+/R3K5Ms3U3OKTz+KnAt39pTif8p9+YLevtHJEPNRSE6LQMO21bd3DSakuVL
Bu0dVvc9OjGUqphDm0yADfVxlkRh1UKJumnV3DI8kTg25EMz8WHUxlVk/EbRJOfM
QW2pht7P1b4zXSzmw+bLTSx513a9duMX8mP8sjaQGRazTzWr+Wait41LYCgsxJka
vm0cV5Y2PBtjFhXng6kLvLpC/yC438bdJ254/6Bd2MA+Q5SgozBa4Pve/RPKpnkX
WxKwGIfoBbNlbqr5HPLj4Mk4w41VTe4GAaCQO8oUr9vshzK6HQYV9mqaZoTxvv4S
UvWVtVB/8C+BG7gcJp5yI8l8m0S1I8pldgFjGXssniU4BVDzuxibrdnHMFdwegH8
J7n5qOEsUBYRJzj3bR3NzYHr5sZrb4f2WHpmYxk7vb/lOecenZFPEH5cGHq+wJGD
JXpOWIT5PdXyuSFdta0mrIVw+IwSZ0KKOQOaQPW3jZDQSLO4W7WPCprLgMH2yocQ
DhhzABEBAAGJAiUEGAEKAA8FAlWcruECGwwFCQhwyoAACgkQ2OQ2RZLzp0l4NxAA
m1V+ZAbK72xuLTD8nx5BH38vW1KdcC5Myp3afKNQtGMPzIcD4Nq3FJsOa0BSuzyC
78H4NvYzSa531a/34Ade/c3s9WQZH/uqrboYtnMXeV/WAfKi6ltGAmsAUA3Ccqup
bpw6nPTJ82MxseudSEbU9xwYbvjd8w5T3S+cpQk20hYmiemFPHedSCO2CPCYW7jr
Vp7dQf3L1LUMLz0XpHpocyhxLUlHVQT0nV2gV6hmn7BmOdHUuj2KnCJCPGO7trlw
ZpbJf7cgv/tF1D9nwvHpwAERxcKS+4hxsu9x1JhFfrfuR/A7epWUS0Mjb6OreEzf
esZMeHmDCbsxS+Z9ADHow3NjHicVIoMvRAh6eJVqbzzGuamfVwTl0zDjZr4nGjsU
X+TMJYArpVLw4KUQMtD6uuliFNMg107tqpWrhXKncSQ5e2Ggduji6rXaeRgD7ZCE
JWmuIOflCvb71wwPadMcUE64QfpJ3nyrbVN4grPvPJbdiY+Ua0Eq3BHRcDvD1cnD
NfPMvT1u6SWWE8IaQ7W+ibVe9DDfvVq1OTRXH/fNo92vdBSrhciTle6LROSdTsxM
R/uctnkU9YsJ3moxQx9qGVgJAbYgFe4jP6EsMQPjPUI2ZTVpdqnLefyGzrS7N2Nt
rJYlwEwPsnPHKhy2ILkwSNzIDRZuqttjD3vbCmh6anI=
=Ls7/
-----END PGP PUBLIC KEY BLOCK-----

**Comment by matheweis** [comment]
The same key with the top two lines of the block removed (the comment and the empty line) will yield the following error: "No packets; cannot extract a key"

**Comment by oconnor663** [comment]
You guys are referring to two different types of comments here:

1) Comments on identities, e.g. `Tor Browser Developers (signing key) <torbrowser@torproject.org>`
2) Comments at the start of ascii-armored exported keys, e.g. `Version: GnuPG v2`

I think you can do whatever you want with (1). I don't know what the rules are with (2), though I wouldn't be surprised if different implementations did different things.
